Makefile

```diff
@@ -76,6 +76,9 @@ docs-and-schema: ## Generate the Zarf Documentation and Schema
 	hack/gen-cli-docs.sh
 	ZARF_CONFIG=hack/empty-config.toml hack/create-zarf-schema.sh
 
+lint-packages-and-examples: build-cli ## Recurisvely lint all zarf.yaml files in the repo except for those dedicated to tests
```

Comment:
Added a build-cli-for-system make command, and rely on that. Though it will only every create windows amd, linux amd, mac-intel, mac-apple since those are the only binaries we set zarf_bin atm